src/std_impls/value_types.rs

```diff
@@ -2,6 +2,8 @@ use std::collections::hash_map::RandomState;
 use std::ptr::drop_in_place;
 use std::time::{Duration, Instant};
 
+/// mark as value type
+#[macro_export]
```

Comment:
I'd prefer not to export this / keep it as a private implementation detail. This "value type" impl is really only supposed to be applied to the Rust primitives, and I'd like to be able to make changes to it under that assumption. End consumers should either use the derive, or write their own macro